commec/screen.py

```diff
@@ -245,7 +245,7 @@ def screen_nucleotides(self):
         """
         # Only screen nucleotides in noncoding regions
         fetch_noncoding_regions(self.databases.protein_db.out_file,
```

Comment:
hm, I wonder if we want to give the out_file as an output of the `.screen()` method for each tool in future?

Reply:
Hmmm could do it as a qol feature for devs I suppose. However it ties the screen call, and the validity of the output to the call, whereas checking files allows for parallelism if desired as the .screen() ultimate just calls subprocesses.